pyproject.toml

```
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["jupyter_packaging~=0.7.7", "setuptools>=40.8.0", "wheel"]
+requires = ["jupyter_packaging~=0.7.7", "jupyterlab>=3.0.0rc4,==3.*", "setuptools>=40.8.0", "wheel"]
```

It should now be possible to update to:

```suggestion
requires = ["jupyter_packaging~=0.7.9", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
```